src/components/Navbar/Navbar_Items/NavbarIconList.tsx

```diff
@@ -0,0 +1,44 @@
+import { CiBadgeDollar, CiCalendarDate, CiCircleCheck, CiHome } from 'react-icons/ci';
+import { PiChartLineUpThin, PiFolderSimpleUserThin } from 'react-icons/pi';
+import { useNavigate } from 'react-router-dom';
+import { Flex, Icon } from '@chakra-ui/react';
+
+export const NAV_LIST = [
+  { name: 'home', icon: CiHome, path: '/' },
```

Comment:
te path sobie weź z tego exportowanego routes co wyżej zasugerowałem, wtedy zmiana tam spowoduje też zmianę tutaj

Reply:
zrobione